eks-worker-al2-variables.json

```diff
@@ -22,7 +22,7 @@
     "pause_container_version": "3.5",    
     "pull_cni_from_github": "true",
     "remote_folder": "",
-    "runc_version": "runc-1.1.4-1.amzn2",
+    "runc_version": "1.1.4-1.amzn2",
```

Comment:
Did you try building an AMI?

Reply:
`make 1.24` succeded, and I can see the proper `runc` is being installed:
```
2022-12-13T11:27:01-08:00:     amazon-ebs: Resolving Dependencies
2022-12-13T11:27:01-08:00:     amazon-ebs: --> Running transaction check
2022-12-13T11:27:01-08:00:     amazon-ebs: ---> Package runc.x86_64 0:1.1.4-1.amzn2 will be installed
2022-12-13T11:27:01-08:00:     amazon-ebs: --> Finished Dependency Resolution
2022-12-13T11:27:01-08:00:     amazon-ebs:
2022-12-13T11:27:01-08:00:     amazon-ebs: Dependencies Resolved
2022-12-13T11:27:01-08:00:     amazon-ebs:
2022-12-13T11:27:01-08:00:     amazon-ebs: ================================================================================
2022-12-13T11:27:01-08:00:     amazon-ebs:  Package     Arch          Version               Repository                Size
2022-12-13T11:27:01-08:00:     amazon-ebs: ================================================================================
2022-12-13T11:27:01-08:00:     amazon-ebs: Installing:
2022-12-13T11:27:01-08:00:     amazon-ebs:  runc        x86_64        1.1.4-1.amzn2         amzn2extra-docker        2.9 M
2022-12-13T11:27:01-08:00:     amazon-ebs:
2022-12-13T11:27:01-08:00:     amazon-ebs: Transaction Summary
2022-12-13T11:27:01-08:00:     amazon-ebs: ================================================================================
2022-12-13T11:27:01-08:00:     amazon-ebs: Install  1 Package
2022-12-13T11:27:01-08:00:     amazon-ebs:
2022-12-13T11:27:01-08:00:     amazon-ebs: Total download size: 2.9 M
2022-12-13T11:27:01-08:00:     amazon-ebs: Installed size: 9.4 M
2022-12-13T11:27:01-08:00:     amazon-ebs: Downloading packages:
2022-12-13T11:27:01-08:00:     amazon-ebs: runc-1.1.4-1.amzn2.x86_64.rpm                              | 2.9 MB   00:00
2022-12-13T11:27:01-08:00:     amazon-ebs: Running transaction check
2022-12-13T11:27:01-08:00:     amazon-ebs: Running transaction test
2022-12-13T11:27:01-08:00:     amazon-ebs: Transaction test succeeded
2022-12-13T11:27:01-08:00:     amazon-ebs: Running transaction
2022-12-13T11:27:02-08:00:     amazon-ebs:   Installing : runc-1.1.4-1.amzn2.x86_64                                    1/1
2022-12-13T11:27:02-08:00:     amazon-ebs:   Verifying  : runc-1.1.4-1.amzn2.x86_64                                    1/1
2022-12-13T11:27:02-08:00:     amazon-ebs:
2022-12-13T11:27:02-08:00:     amazon-ebs: Installed:
2022-12-13T11:27:02-08:00:     amazon-ebs:   runc.x86_64 0:1.1.4-1.amzn2
```